vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -99,10 +99,9 @@ $global:imagesToPull += @(
     "mcr.microsoft.com/oss/kubernetes-csi/azurefile-csi:v1.30.1-windows-hp", # for k8s 1.29.x
     # Addon of Azure secrets store. Owner: jiashun0011 (Jiashun Liu)
     "mcr.microsoft.com/oss/kubernetes-csi/secrets-store/driver:v1.3.4",
-    "mcr.microsoft.com/oss/kubernetes-csi/secrets-store/driver:v1.4.1",
```

Comment:
Please keep the old version if the old version is still used for now.

Reply:
updated.